### DISCUSSION TITLE
Country Selectbox in 2FA does not include Turkey.

### DISCUSSION THREAD

**Discussion by emrtnm** [community/community#65455] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello. I am from Turkey. I want to use 2FA for my account. I click this page:https://github.com/settings/two_factor_authentication/setup/intro and I select SMS Authentication. But Selectbox does not include my country(Turkey). I want to choose Turkey(+90) but I dont it. Can you add our Country to selectbox ? Thanks :)

**Comment by ilhan-athn7** [comment]
I have the same problem.

**Comment by ilhan-athn7** [comment]
![](https://user-images.githubusercontent.com/80551220/265594274-47fb21cb-c325-4495-9ae6-4191cb2f5449.png)
